### ISSUE TITLE
Include older version of NBMiner with 0.65% fee

### ISSUE THREAD

**Issue by samdave123** [RainbowMiner/RainbowMiner#977]
I'm not sure if this has already been considered, but I think it makes sense to include the old NBMiner with 0.65% fee? Profitability may be higher than the new miner with higher dev fee.

**Comment by RainbowMiner** [comment]
Yes, potentially. I will try to cross-evaluate the Ethash mining.
